041_llxc_L777.py

```
@@ -0,0 +1,940 @@
+    """Unarchive LXC container"""
+    print (_(" * Unarchiving container: %s..." % (CONTAINERNAME)))
+    cont = lxc.Container(CONTAINERNAME)
+    if cont.defined:
+        print ("   %serror:%s a container by name %s already exists unarchived"
+               % (RED, NORMAL, CONTAINERNAME))
+        exit(1)
+    requires_container_nonexistance()
+    # If we're on btrfs we should create a subvolume
+    if is_path_on_btrfs(CONTAINER_PATH):
+        print ("   container path is on btrfs, creating subvolume...")
+        os.popen("btrfs subvolume create " + CONTAINERNAME)
```

Ruff (0.14.5)

Starting a process with a shell, possible injection detected

(S605)